packages/@dcl/inspector/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "@dcl/inspector",
   "version": "0.1.0",
+  "dependencies": {
+    "earcut": "^2.2.4"
```

Comment:
is this being used? If so, let's move it to `devDependencies`

Reply:
removed 👌 (was for a test for 3D text in babylon)